src/planner/plan/plan_read.cpp

```diff
@@ -74,5 +78,19 @@ void QueryPlanner::planInQueryCall(
     }
 }
 
+void QueryPlanner::planLoadFrom(
+    binder::BoundReadingClause* readingClause, std::vector<std::unique_ptr<LogicalPlan>>& plans) {
+    auto loadFrom = reinterpret_cast<BoundLoadFrom*>(readingClause);
+    for (auto& plan : plans) {
+        if (!plan->isEmpty()) {
+            auto tmpPlan = std::make_unique<LogicalPlan>();
+            appendScanFile(loadFrom->getInfo(), *plan);
+            appendCrossProduct(AccumulateType::REGULAR, *plan, *tmpPlan);
```

Comment:
What does L86-88 mean here? Why are we appending cross product between ScanFile and an empty tmpPlan?